psydac/api/ast/nodes.py

```diff
@@ -166,7 +223,7 @@ class EvalField(BaseNode):
        This function computes atomic expressions needed
        to evaluate  EvaluteField/VectorField final expression
     """
-    def __new__(cls, atoms, q_index, l_index, q_basis, l_basis, coeffs, l_coeffs, g_coeffs, tests, mapping, pads, nderiv, mask=None):
+    def __new__(cls, atoms, q_index, l_index, q_basis, l_basis, coeffs, l_coeffs, g_coeffs, tests, mapping, nderiv, mask=None):
```

Comment:
Since this is a very important class, I think it is a good time for adding a `Parameters` section to its docstring!

Reply:
done